src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
```

Comment:
let's create constant NUMBER_OF_BALLS for number 3 (because it is a magic number)
more info about constants - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names